### ISSUE TITLE
Error: Response is constant across permutations. Please check your model

### ISSUE THREAD

**Issue by JiaxiangBU** [thomasp85/lime#151]
LIMI seizes me, because of its good display way.
I try to use LIME to explain Xgboost Model. for text classification topic, I find there are a lot of successful minimal examples online, such as [this one](https://shirinsplayground.netlify.com/2018/07/explaining_ml_models_code_text_lime/).
I reproduce it successfully.

However, when I apply lime functions on a chinese-character dataset, I always get this error message.

```
Error: Response is constant across permutations. Please check your model
```

It seemingly means that the target variable is constant. I double check it on the origin function.
See it [here](https://github.com/thomasp85/lime/blob/15612fab7ac5e34f2a049e5b53483d67321290cb/R/lime.R)

But in my dataset, my target variable is a binary variable instead of a constant one.

For reproducible purpose, I upload the dataset and minimal exmaple code here.

1. [bad_comment.csv](https://jiaxiangbu.github.io/learn_nlp/text-classification/datasets/bad_comment.csv)
1. [good_comment.csv](https://jiaxiangbu.github.io/learn_nlp/text-classification/datasets/good_comment.csv)
1. [stopwords.txt](https://jiaxiangbu.github.io/learn_nlp/text-classification/datasets/stopwords.txt)
1. [lime-issue.R](https://jiaxiangbu.github.io/learn_nlp/text-classification/lime-issue.R)

The last code of `.R` file produces the error in the title.

**Comment by JiaxiangBU** [comment]
I find this kind of error results from the structure input. 
Here is my test results.

```
test_comment_1 <- "卧槽，这酒也太好喝了，简直没的说，绝对的物有所值，下来再买"
lime::explain(test_comment_1,explainer,n_labels = 1,n_features = 5) %>% plot_features
```
I get the successful output.

But the next one,

```
test_comment_2 <- "红红火火"
safely(lime::explain)(test_comment_2,explainer,n_labels = 1,n_features = 5)
```

```
## $result
## NULL
## 
## $error
## <simpleError: Response is constant across permutations. Please check your model>
```

I think the input includes duplicated words `红` and `火`.

**Comment by BrazilForever11** [comment]
I got the same issue, I am guessing this is due to class imbalance. When you pick value from the test set, lime finds neighborhood around this value in train set and using permutation picks value from this neighborhood. And due to class imbalance all values come out to be of the same class, hence the error.
